crates/deskulpt/capabilities/canvas.json

```
@@ -4,5 +4,5 @@
   "description": "Capabilities of the canvas window.",
   "local": true,
   "windows": ["canvas"],
-  "permissions": ["core:event:default"]
+  "permissions": ["core:event:default", "opener:default"]
```

```suggestion
  "permissions": [
    "core:event:default",
    "opener:allow-open-url",
    "opener:allow-default-urls"
  ]
```

> For some reason, if I only add the permission `opener:allow-default-urls`, which should allow opening urls with default browser app, the hyperlink won't be opened at all. So I have to use the permission `opener:default`.

This is because you are missing the `opener:allow-open-url` permission. You may check its [initialization script](https://github.com/tauri-apps/plugins-workspace/blob/a6b854032d0b10f0f17c4ffa6bdf4a05429e05fb/plugins/opener/guest-js/init.ts), where under the hood it calls `plugin:opener|open_url`.

---

Thanks for the information. Fixed.